WORKSPACE

```diff
@@ -14,62 +13,83 @@ android_sdk_repository(
     api_level = 28,
 )
 
+RULES_ANDROID_TAG = "0.1.1"
+
+RULES_ANDROIDL_SHA = "cd06d15dd8bb59926e4d65f9003bfc20f9da4b2519985c27e190cddc8b7a7806"
+
+http_archive(
```

Comment:
Without the addition of this, we are getting error in the binary build.
Reference - https://github.com/bazelbuild/buildtools/issues/785
```
ERROR: Skipping '//:oppia': error loading package '': Unable to find package for @rules_android//android:rules.bzl: The repository '@rules_android' could not be resolved.
WARNING: Target pattern parsing failed.
Analyzing: 0 targets (1 packages loaded)
ERROR: error loading package '': Unable to find package for @rules_android//android:rules.bzl: The repository '@rules_android' could not be resolved.
```

Reply:
I'm assuming this is because we're now importing android_binary?

Reply:
Yes, I guess, can't say surely.